core/templates/pages/contributor-dashboard-admin-page/contributor-admin-dashboard-page.component.ts

```diff
@@ -76,6 +68,47 @@ export class ContributorAdminDashboardPageComponent implements OnInit {
   checkMobileView(): boolean {
     return (this.windowRef.nativeWindow.innerWidth < 800);
   }
+
+  updateSelection(selection: string): void {
+    this.selection = selection;
+    this.setActiveTab(selection);
```

Comment:
In updateSelection are we just setting activeTabName ? If yes why not just call `this.setActiveTab` directly instead of creating a new method here, or is there anything I m missing here ?